### PR TITLE
Redone TMA script.

### PR DESCRIPTION
**Pull request by FakelsHub**

### CODE REVIEW

**Comment by Lexx2k** [comment]
Did some tests quick tests. Looks good so far. Only the font is wrong / borked, and the interface graphic is shifted a pixel or two to the right.

**Comment by FakelsHub** [comment]
That you have with fonts they not original?
so fonts look at me
tell

**Comment by Lexx2k** [comment]
I am using the fonts that are currently on the repo in /data/.
For a test I've deleted them. [This](https://i.imgur.com/hw9mECA.png) is what it looks like now.

**Comment by FakelsHub** [comment]
> Why use sfall_func1 first and then draw_image()? Isn't it pretty much the same?

just fewer parameters.

File: Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl [hunk 1]
```diff
@@ -148,153 +187,151 @@ procedure create_button begin
...
+      Format("Ask about what ?", 240, 70, 200, 30, 0);
+      Format("Done", 230, 118, 100, 30, 0);
+      Format("Cancel", 355, 118, 100, 30, 0);
 
-      create_win("history_block", (Scr_Width - 310), (Scr_Height + 153), 60, 30);
-      SelectWin("history_block");
-      draw_image("PCX/block_history.frm", 0, 0, 0, 0);
+      SetFont(100);
+      SetTextColor(0.0, 1.0, 0.0);
+      Format("$" + dude_caps, 4, 36, 70, 15, justifycenter);
```
Comment (FakelsHub) [inline Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: Choose the font here, try to SetFont (1)

Comment (Lexx2k) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: SetFont(1) looks much closer, but it's still not the same. Seems it is missing smooth edges. Also [this](https://i.imgur.com/ZCy4VSO.png) happens after closing the interface and moving the mouse over dialog options.

Comment (FakelsHub) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: try 2, 3

Comment (FakelsHub) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: or 101

Comment (Lexx2k) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: 101 is the correct font. Just have to tweak the xy text coordinates now to make them fit with the original. Also seems like the gray window background is visible in a few places - you can see them [here](https://i.imgur.com/bRUK41D.png) pretty clearly if you quickly enable/disable the tma interface.

Comment (Lexx2k) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: Fixed the bottom right grey line with changing `create_win("tellme_interface", (Scr_Width - 320), Scr_Height, 640, 190);` << it was 641 before (on purpose?)

Comment (Lexx2k) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: Oh, and I found one more barter-button issue. :/ Your fix works when clicking the button on the interface, but if the player uses a dialog option to go into barter, the tma-button remains visible as before.

**Comment by FakelsHub** [comment]
> Fixed the bottom right grey line with changing create_win("tellme_interface", (Scr_Width - 320), Scr_Height, 640, 190); << it was 641 before (on purpose?)

try to set 640 if the game does not crash
(this is some bug in the engine. I think I'll fix it)

> Oh, and I found one more barter-button issue. :/

Yes exactly, I forgot about it, but you have to correct it yourself :)

**Comment by FakelsHub** [comment]
Also seems like the gray window background is visible in a few places - you can see them here pretty clearly if you quickly enable/disable the tma interface.

change the fill color to black

**Comment by Lexx2k** [comment]
> Yes exactly, I forgot about it, but you have to correct it yourself :)

But how? Any special way, or should we add some code to every barter procedure that deletes the button? (In that case, wouldn't it be better to hook something into `gdialog_mod_barter()` or so?

**Comment by FakelsHub** [comment]
> But how? 

This is a difficult question. :(
Make a simple delete after the animation for now.

File: Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl [hunk 2]
```diff
@@ -148,160 +189,153 @@ procedure create_button begin
...
+      fetch_input := true;
       input := "";
       //debug("TMA input enabled!");
 
-      // Opening the interface clears the Reply window.
-      dialog_message("");
+      // Reply window (clear the Reply window)
+      create_win("tellme_reply", (Scr_Width - 190), (Scr_Height - 64), 390, 64);
+      SelectWin("tellme_reply");
+      Display("PCX/tellmeabout_reply.pcx");
```
Comment (FakelsHub) [inline Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: Replace all `/` to `\\` in paths

Comment (Lexx2k) [reply Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl]: Ok, done.

**Pull request merged**